rover/start/tmuxprocmon

```diff
@@ -9,6 +9,7 @@
 
 source $HOME/.bashrc
 source $HOME/owr_software/rover/devel/setup.bash
+# source $HOME/ros_can_nodes/devel/setup.bash
```

Comment:
fyi if you're having problems with this these source files actually overide each other. To use multiple source one, build the other package and then source the package you just built (and repeat if you want more of them) 
Having them in here is also really annoying because it means tmux won't use what's in your current bash session (notice I deleted ln11 in my slam pr for exactly that reason)

Reply:
Yep, I was indeed having problems with that.
I am not sure why you deleted that line, because now it means that we have to source the `setup.bash` file separately before running tmux.

Reply:
(suggestion) It should be in .bashrc  so you can define the correct order, etc :P 
either way don't leave it commented out

@simonireland was also asking questions about a problem with the same cause for optical flow